config/registry/schemas/modules/custom-terraform.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$id": "https://app.runx.dev/modules/custom-terraform",
+  "description": "To bring in your own custom terraform module",
+  "type": "object",
+  "properties": {
+    "path_to_module": {
+      "type": "string",
+      "description": "The path to your terraform module",
+      "default": null
+    },
+    "terraform_inputs": {
+      "type": "object",
+      "description": "The variables which you wish to pass into your custom module.",
+      "default": {}
+    },
+    "type": {
+      "description": "The name of this module",
+      "enum": [
+        "custom-terraform"
+      ]
+    }
+  },
+  "opta_metadata": {
+    "module_type": "env",
+    "clouds": [
+      "aws",
+      "gcp"
+    ]
+  },
+  "required": [
+    "path_to_module",
+    "type"
+  ]
+}
```

Comment:
I see I have not removed the config/schemas/modules directory yet, doing that in another PR. 

This file need not be put into  config/registry/schemas/modules/, the single copy in modules/custom_terraform is adequate.